<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRegelaarInstellingenDialoog</class>
 <widget class="QDialog" name="QtRegelaarInstellingenDialoog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>650</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>562</width>
     <height>573</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" rowminimumheight="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
    <item row="9" column="0" colspan="4">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="minSnelheid">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="eloc">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Eloc</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>locType</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLineEdit" name="afstand2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="topSnelheid"/>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Clip stoppen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>topSnelheid [km/h]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="clipStoppen"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="totaalTijd"/>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="clipRijden"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>lengte [cm]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Alpha rijden</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="alphaStoppen"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lengte"/>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>totale tijd [hh:mm]</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Apha Stoppen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="alphaRijden">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="errors">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Clip rijden</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="totaalAfstand"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>totale afstand [m]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Errors</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="stand2"/>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="19" column="3">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>last regeling gain1</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="4">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="locType"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="stand1"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>maxSnelheid [tics]</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>minSnelheid [tics]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="maxSnelheid"/>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>programma naam</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Stand2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="helling"/>
    </item>
    <item row="11" column="3">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Afstand2 [cm]</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Stand1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="4">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="afstand1"/>
    </item>
    <item row="14" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Dode tijd</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLineEdit" name="programmaNaam"/>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="kLpf"/>
    </item>
    <item row="13" column="3">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>PlusMinus</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>helling</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QLineEdit" name="dodeTijd"/>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Afstand1 [cm]</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QLineEdit" name="plusMinus"/>
    </item>
    <item row="16" column="2" colspan="2">
     <widget class="QCheckBox" name="herlaadProgramma">
      <property name="text">
       <string>herlaad programma</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLineEdit" name="langzaam"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>kLpf</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="17" column="3">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>rijden</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QLineEdit" name="rijden"/>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>langzaam</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="lastStand1"/>
    </item>
    <item row="19" column="2">
     <widget class="QLineEdit" name="lastGain1"/>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>last regeling stand1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QCheckBox" name="laatsteWagonCheck">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Laatste wagon check</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="4">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>last regeling stand2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QLineEdit" name="lastStand2"/>
    </item>
    <item row="20" column="2">
     <widget class="QLineEdit" name="lastGain2"/>
    </item>
    <item row="20" column="3">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>last regeling gain2</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>last regeling keuze</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QLineEdit" name="lastRegelingKeuze"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QtRegelaarInstellingenDialoog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QtRegelaarInstellingenDialoog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
